### PR TITLE
Fix ToS link to point to correct file

### PR DESCRIPTION
**Pull request by caesar2164**
- the external adminguide file moved, this updates link

@stvstnfrd & @mondiaz

### CODE REVIEW

**Comment by mondiaz** [comment]
@caesar2164 that looks good to me. thanks!

File: templates/static_templates/theme-tos.html [hunk 1]
```diff
@@ -477,7 +477,8 @@ <h2 id="tos">${_('Terms of Service for Stanford Online Courses')}</h2>
             servicemarks appearing on the Site are the marks of their respective owners. Information about Stanford
             trademarks may be found at {trademark_link}.
         ''').format(
-            trademark_link='<a href="http://adminguide.stanford.edu/15_5.pdf">http://adminguide.stanford.edu/15_5.pdf</a>'
+            trademark_link='<a href="https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4">
+                https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4</a>'
```
Comment (stvstnfrd) [inline templates/static_templates/theme-tos.html]: See comment on suclass PR about reformatting this string.

File: templates/static_templates/theme-tos.html [hunk 2]
```diff
@@ -477,8 +477,10 @@ <h2 id="tos">${_('Terms of Service for Stanford Online Courses')}</h2>
             servicemarks appearing on the Site are the marks of their respective owners. Information about Stanford
             trademarks may be found at {trademark_link}.
         ''').format(
-            trademark_link='<a href="https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4">
-                https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4</a>'
+                trademark_link = (
```
Comment (stvstnfrd) [inline templates/static_templates/theme-tos.html]: see other comment about kwargs

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**